metricbeat/docs/modules/system/core.asciidoc

```diff
@@ -18,28 +18,30 @@ Below is an example document generated by this metricset.
 [source,json]
 ----
 {
-    "@timestamp":"2016-05-11T07:14:06.329Z",
-    "beat":{
-        "hostname":"ruflin",
-        "name":"ruflin"
-    },
-    "metricset":"core",
-    "module":"system",
-    "rtt":135,
-    "system-core":{
-        "core":0,
-        "idle":26880742,
-        "iowait":0,
-        "irq":0,
-        "nice":0,
-        "softirq":0,
-        "steal":0,
-        "system":7499682,
-        "system_p":0,
-        "user":9354158,
-        "user_p":0
-    },
-    "type":"metricsets"
+  "@timestamp": "2016-05-25T20:49:40.557Z",
+  "beat": {
+    "hostname": "mar.local",
```

Comment:
I propose we use the same host in each of these examples. How about `host.example.com`?